src/main/java/org/launchcode/liftoffproject/models/Chore.java

```diff
@@ -85,14 +85,27 @@ public void setRewardPoints(int rewardPoints) {
         this.rewardPoints = rewardPoints;
     }
 
-    public String getDueDay() {
+    public DayOfTheWeek getDueDay() {
         return dueDay;
     }
 
-    public void setDueDay(String dueDay) {
+    public void setDueDay(DayOfTheWeek dueDay) {
         this.dueDay = dueDay;
     }
 
+    // Enum for the days of the week
+    public enum DayOfTheWeek {
+        SUNDAY,
+        MONDAY,
+        TUESDAY,
+        WEDNESDAY,
+        THURSDAY,
+        FRIDAY,
+        SATURDAY
+    }
+
```

Comment:
I don't think you should need this code if you import the `DayOfTheWeek.java` class, because the enum is defined there.
```suggestion
```

Reply:
Thanks for going through the code. I will work on the fixes today and you can look at it later. I do appreciate your input.